imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -69,16 +71,16 @@ def process_and_write_data(
     ----------
     packet_data: list[MagL0]
         List of MagL0 packets to process, containing primary and secondary sensor data
-    raw_attrs
+    raw_attrs: dict
```

Comment:
```suggestion
    raw_attrs : dict
```

Reply:
Is this something we all should be doing? Because Many of us are not.

Reply:
I believe the spaces between the colon is part of the numpydoc convention which we try to follow (example [here](https://numpydoc.readthedocs.io/en/latest/format.html#parameters)). That being said, I think it is a pretty minor formatting issue.

Reply:
I can do that in another PR since we are gonna need to change in many places. What do you think?

Reply:
I think you can probably hold off for now, as we might add some automated checks for this (https://github.com/IMAP-Science-Operations-Center/imap_processing/issues/574). We could fix these when that is implemented.

Reply:
Ah ok. cool